From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@snobu__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@snobu - Thanks!

vladvino committed:
Edits accounting for 12/1/2017 pricing change

fauhse committed:
Adding firewall and proxy configuration article to Azure File Sync docs

fauhse committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into afs-R1

fauhse committed:
Minor style issues

fauhse committed:
Improving from automated feedback

fauhse committed:
More improvements.

fauhse committed:
feedback from Will and tool

fauhse committed:
Adressing style issues

fauhse committed:
Added "Next steps" link to storage sync planning doc

fauhse committed:
grammar and regional endpoint URLs

fauhse committed:
spelling

fauhse committed:
Fixing the broken link

fauhse committed:
removing repeated phrase

fauhse committed:
Fixing layout issues around the second table

fauhse committed:
small fixes

fauhse committed:
incorporating Will's feedback

fauhse committed:
replacing a rather strange phrase

fauhse committed:
Added User Voice part

Lisaco88 committed:
edit pass: how-to-connect-mxchip-iot-devkit

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into how-to-connect-mxchip-iot-devkit

Lisaco88 committed:
edit pass: how-to-connect-mxchip-iot-devkit

rjagiewich committed:
edit pass: how-to-configure-server-logs-2

cartermp committed:
Move sample link up

cartermp committed:
Acrolynx feedback

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into how-to-connect-mxchip-iot-devkit

rjagiewich committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into how-to-configure-server-logs-2

rjagiewich committed:
edit pass: how-to-configure-server-logs-2

rjagiewich committed:
edit pass: how-to-configure-server-logs-2

rjagiewich committed:
edit pass: how-to-configure-server-logs-2

cartermp committed:
Feedback from the team

cartermp committed:
Clean up

cartermp committed:
Merge branch 'master' into mongo-qs

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into how-to-connect-mxchip-iot-devkit

Lisaco88 committed:
edit pass: how-to-connect-mxchip-iot-devkit

rjagiewich committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into how-to-configure-server-logs-2

rjagiewich committed:
edit pass: how-to-configure-server-logs-2

rjagiewich committed:
edit pass: how-to-configure-server-logs-2

genlin committed:
update the steps about increase qoute

genlin committed:
update the steps

fauhse committed:
All Tamra feedback

Commit:
first cut of PFX password updates

Commit:
grammar fix

cartermp committed:
Remove mention of emulator

cartermp committed:
Merge branch 'mongo-qs' of https://github.com/cartermp/azure-docs-pr into mongo-qs

cartermp committed:
Update ms.date

Commit:
couple more edits

divyaswarnkar committed:
c1

divyaswarnkar committed:
c2

MSSedusch committed:
SAP: add NFS guide

MSSedusch committed:
SAP: add manual deployment steps

MSSedusch committed:
fix spelling errors

MSSedusch committed:
fix drbd config

MSSedusch committed:
fix intend

mimig1 committed:
Added metadata to include file.

cartermp committed:
Merge pull request #3 from mimig1/patch-226

Added metadata to include file.

divyaswarnkar committed:
c3

divyaswarnkar committed:
c4

Commit:
automation: startstop tutorial update

tamram committed:
adding AFS firewall and proxy settings to TOC

v-nagta committed:
USER STORY 1214425 SaaS App Tutorial: ADP Update

Commit:
updates

Commit:
resized images

Commit:
updates to images

Commit:
correcting links

Commit:
corrected bookmark and adding information about shutdown only

billmath committed:
adding note

billmath committed:
updating

billmath committed:
updating

billmath committed:
updating

billmath committed:
updating

divyaswarnkar committed:
c6

divyaswarnkar committed:
c7

divyaswarnkar committed:
c8

fauhse committed:
Merge pull request #1 from tamram/tamram-34211

adding AFS firewall and proxy settings to TOC

vladvino committed:
Client cert auth in send request

curtand committed:
[Azure AD enterprise apps] changed name to current owner

mawah committed:
Insert quotation markers around passwords in az commands

Per LiveFyre feedback

swkrish committed:
Update active-directory-aadconnect-sso.md

v-geberr committed:
concept collaborator

v-geberr committed:
edits

mattbriggs committed:
adding note

v-geberr committed:
toc and link fixes

v-geberr committed:
edits

Commit:
username length

mgreenegit committed:
address json examples

v-geberr committed:
edits

mgreenegit committed:
name field

christiankuhtz committed:
remove limit

vladvino committed:
Fixes, metadata and more limits

swkrish committed:
Update active-directory-aadconnect-sso-faq.md

v-geberr committed:
edit

tfitzmac committed:
fixes MicrosoftDocs/azure-docs#5947

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-current-limitations.md

KaitlynCorazao committed:
Permissions + remove SQL DB

Changed permissions model so Owner is not required to view blade first.
- removed note
- modified FAQ

Removed SQL elastic pool recommendation
- remove SQL DB from FAQ on what resources we have recs for

mgreenegit committed:
missing bracket

mattbriggs committed:
update link

Commit:
Merge pull request #36066 from cynthn/linux-faq

username length

KaitlynCorazao committed:
Remove SQL Elastic Pool Rec

Removing section on SQL Elastic Pool since the recommendation was removed.

mmacy committed:
[ACR] service limits fix

vladvino committed:
Parse date flag in JSON to XML

rjagiewich committed:
Merge pull request #36059 from v-geberr/0322-concept-collab

concept collaborator

PRMerger13 committed:
Merge pull request #36078 from kasparks/patch-3

Permissions + remove SQL DB

barbkess committed:
consolidate backup and restore

PRMerger15 committed:
Merge pull request #36077 from swkrish/master

Added DSSO tenanted endpoint

barbkess committed:
fix merge conflict

rjagiewich committed:
Merge pull request #33431 from vladvino/master

Edits accounting for 12/1/2017 pricing change

rjagiewich committed:
Merge pull request #36085 from mmacy/acr-limits-fix

[ACR] service limits fix

Commit:
added note for no WSL support

tfitzmac committed:
changed formatting

barbkess committed:
updates

barbkess committed:
fix next steps

PRMerger12 committed:
Merge pull request #36062 from mattbriggs/mdb_20180322_psquick

adding note

PRMerger15 committed:
Merge pull request #36074 from tfitzmac/0322move

added required permissions

zhiweiwangmsft committed:
update powershell cmdlet questions

zhiweiwangmsft committed:
update wording

CarlRabeler committed:
Update sql-database-managed-instance.md

Commit:
Merge pull request #36083 from kasparks/patch-4

Remove SQL Elastic Pool Rec

divyaswarnkar committed:
c8

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sf-win-container-qs-fix
sync with upstream master

Commit:
Merge pull request #36099 from CarlRabeler/patch-35

Update sql-database-managed-instance.md

zhiweiwangmsft committed:
remove typo

barbkess committed:
remove two manage files

PRMerger15 committed:
Merge pull request #36098 from zhiweiwangmsft/patch-23

update powershell cmdlet questions

roygara committed:
Fixing incorrect link.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into bugfix

barbkess committed:
fix link

PRMerger18 committed:
Merge pull request #36104 from roygara/bugfix

Storage, fixing python SDK link

barbkess committed:
remove reference overview

barbkess committed:
fix broken links

charwen committed:
Added multiple performance metrics

PRMerger18 committed:
Merge pull request #36105 from charwen/patch-49

Added multiple performance metrics

mrbullwinkle committed:
Changes to address GitHub Issue #6108

Commit:
correcting redirect

chaosrealm committed:
Document MongoDB support

rjagiewich committed:
Merge pull request #35131 from cartermp/mongo-qs

[CosmosDB Mongo API Quickstart] Shorten some prerequisites

j-martens committed:
AML: Metadata updates

rjagiewich committed:
Merge pull request #35691 from JimacoMS2/sf-win-container-qs-fix

Update container quickstart with PFX password and alternative cluster creation note

rjagiewich committed:
Merge pull request #35823 from divyaswarnkar/liquid

Liquid

Commit:
Merge pull request #36109 from j-martens/master

AML: Metadata updates

j-martens committed:
AML: Metadata updates

Commit:
Merge pull request #36106 from mrbullwinkle/mrb_03_22_2018_javascript

Changes to address GitHub Issue #6108

Commit:
minor spelling mistakes

Commit:
minor spelling mistakes

rjagiewich committed:
Merge pull request #36079 from mgreenegit/mgreenegit-extensions-dsc-template

Mgreenegit extensions dsc template

chaosrealm committed:
Fix a warning and simplify Next Steps section

j-martens committed:
AML: Metadata update

j-martens committed:
AML: Metadata updates

rjagiewich committed:
Merge pull request #35900 from georgewallace/autostartstopreview

automation: start stop tutorial update

chaosrealm committed:
Fix a typo

rjagiewich committed:
Merge pull request #36107 from georgewallace/redirectupdate

automation: correcting redirect

rjagiewich committed:
Merge pull request #36103 from barbkess/manage

remove two manage files

Commit:
Fixed bookmark issues

rjagiewich committed:
Merge pull request #36088 from barbkess/restore

consolidate backup and restore

rjagiewich committed:
Merge pull request #36110 from j-martens/master

AML: Metadata updates

PRMerger19 committed:
Merge pull request #36108 from chaosrealm/patch-4

Document MongoDB support

Commit:
removed obsolete link

rjagiewich committed:
Update howto-configure-server-logs-using-cli.md

rjagiewich committed:
Update howto-configure-server-logs-in-cli.md

christiankuhtz committed:
Remove preview

christiankuhtz committed:
Tweak

Commit:
fixed first sentence of note

navyasric committed:
Update instructions for adding certificates

This addresses issue https://github.com/MicrosoftDocs/azure-docs/issues/5494

christiankuhtz committed:
Tweak

PRMerger12 committed:
Merge pull request #36094 from JimacoMS2/sf-get-started-linux-fix

Added note for no WSL support

navyasric committed:
Update steps to find registered App properties

This addresses issue https://github.com/MicrosoftDocs/azure-docs/issues/5550

PRMerger13 committed:
Merge pull request #36115 from christiankuhtz/patch-102

Remove preview

PRMerger16 committed:
Merge pull request #36117 from navyasric/patch-6

Update steps to find registered App properties

ajaycode committed:
region information

Commit:
Merge pull request #36118 from ajaycode/network-monitoring-overview

region information

Commit:
typo

PRMerger12 committed:
Merge pull request #36120 from Blackmist/freshness

typo

MSSedusch committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into sap-nfs

MSSedusch committed:
update article data
reduce resync rate

billmath committed:
fixing url

PRMerger14 committed:
Merge pull request #35935 from billmath/aadc

adding note

christiankuhtz committed:
Correction

Commit:
Merge pull request #36123 from christiankuhtz/patch-103

Correction

Commit:
Merge pull request #36073 from christiankuhtz/patch-100

remove limit

Commit:
Merge pull request #36122 from billmath/qlik

fixing url

Lisaco88 committed:
Merge pull request #35398 from genlin/master301

resource-manager-core-quotas-request.md

christiankuhtz committed:
replaced table

Lisaco88 committed:
Merge pull request #36113 from iRaindrop/vbruham_buildrptcleanup_batchA

Build report cleanup - illegal links - batch A

CarlRabeler committed:
Update sql-database-managed-instance.md

PRMerger17 committed:
Merge pull request #36128 from christiankuhtz/patch-105

replaced table

PRMerger20 committed:
Merge pull request #36056 from mawah/patch-1

Insert quotation markers around passwords in az commands

ktoliver committed:
Merge pull request #35884 from v-nagta/adp

USER STORY 1214425 SaaS App Tutorial: ADP Update

ShawnJackson committed:
Merge pull request #34955 from Lisaco88/how-to-connect-mxchip-iot-devkit

edit pass: how-to-connect-mxchip-iot-devkit

Commit:
Merge pull request #36129 from CarlRabeler/patch-36

Update sql-database-managed-instance.md

ShawnJackson committed:
Update howto-configure-server-logs-in-cli.md

ShawnJackson committed:
Update howto-configure-server-logs-using-cli.md

PRMerger14 committed:
Merge pull request #36081 from curtand/coreapp0322

[Azure AD enterprise apps] changed name to current owner

MSSedusch committed:
add new NFS article to TOC

ktoliver committed:
change ms.date format per guidance

change ms.date format per guidance

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

ktoliver committed:
Merge pull request #36116 from navyasric/patch-5

Update instructions for adding certificates

ShawnJackson committed:
Update howto-configure-server-logs-in-cli.md

ktoliver committed:
Merge pull request #34211 from fauhse/afs-R1

AFS Firewall and Proxy doc

v-alje committed:
Merge pull request #36131 from MicrosoftDocs/FromPublicRepo

From public repo

GitHubber17 committed:
Merge pull request #35762 from MSSedusch/sap-nfs

SAP: add NFS HA on SLES article

ShawnJackson committed:
Merge pull request #35251 from rjagiewich/how-to-configure-server-logs-2

edit pass: How to configure server logs

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
